region/wreckedship/main/Basement.json

```diff
@@ -350,11 +386,37 @@
       ],
       "note": "Use the camera scroll blocks just left of the bomb blocks, in order to overload PLMs and go through them."
     },
+    {
+      "link": [2, 3],
+      "name": "G-mode Morph Power Bombs, Wait for the Workrobot",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"ammo": {"type": "PowerBomb", "count": 2}}
+      ],
+      "note": [
+        "On room entry, dodge the atomic and quickly place a Power Bomb on the left half of the platform, then a second in order to kill both Atomics.",
+        "Placing it on the right half will hit the bomb block and it will no longer be passable.",
```

Comment:
You are safe until about halfway between the platform and the wall.  Where the large foreground structure is.

Reply:
Weird. I tried this 2-3 times and couldnt get them overloaded. Not sure why, but it's working now